### PR TITLE
Add `<progress>` element

### PR DESCRIPTION
**Pull request by petele**

### CODE REVIEW

**Comment by petele** [comment]
@ddbeck - should this get 

```
group:
  - forms
  - html-elements
```

or just `forms`?

File: features/progress.yml [hunk 1]
```diff
@@ -1,8 +1,9 @@
-draft_date: 2024-06-05
 name: <progress>
-description: The `<progress>` element TODO.
+description: The `<progress>` element displays an indicator showing the completion progress of a task, typically displayed as a progress bar.
 spec: https://html.spec.whatwg.org/multipage/form-elements.html#the-progress-element
-group: forms
+group:
+  - forms
+  - html-elements
 caniuse: progress
```
Comment (ddbeck) [inline features/progress.yml:7]: I'm pretty skeptical of the Safari iOS mismatch for the DOM interface and the element but it's very historical and AFAICT impossible to test now. Let's do this to align to caniuse:

```suggestion
caniuse: progress
compute_from: html.elements.progress
```

File: features/progress.yml [hunk 2]
```diff
@@ -1,9 +1,11 @@
...
+compute_from: html.elements.progress
```
Comment (ddbeck) [inline features/progress.yml:8]: Sorry, I messed up the suggestion here, though I'm not sure why no error was thrown on the extra key on the top-level object.

```suggestion
status:
  compute_from: html.elements.progress
```

**Comment by petele** [comment]
@ddbeck - all good, I should have re-run `dist` after your suggestion...

**Review by ddbeck — APPROVED** [review]

**Pull request merged**